test/contribution.test.js

```diff
@@ -1,6 +1,7 @@
 const contribution = require('../src/contribution');
 
 const username = 'jamieweavis';
+const fakeUsername = 'jamieweavisjamieweavis';
```

Comment:
This could _potentially_ exist in the future, although very unlikely 😉

GitHub restricts username length to 39 characters so perhaps we could use a username which is 40 characters long instead, then we can know for sure that the username will never exist - a random string will do.

Also could we change this variable name to `invalidUsername`?

Reply:
Yes you're right, I will change that soon !